session/src/test/java/org/apache/iotdb/session/SessionCacheLeaderUT.java

```diff
@@ -738,6 +1073,8 @@ private void addLine(
 
   static class MockSession extends Session {
 
+    private MockSessionConnection mockSessionConnection;
```

Comment:
The name of this `mockSessionConnection` is misleading, I didn't understand the meaning until after debugging. I think we can change this name to a better one, such as `lastConstructedSessionConnection`

Reply:
have changed